Tools/CPUInfo/CPUInfo.cpp

```diff
@@ -28,8 +31,13 @@ static uint32_t sysclk = 0;
 static EKF_Maths ekf;
 
 HAL_Semaphore sem;
+AP_ESC_Telem telem;
 
 void setup() {
+#if defined(STM32H7)
```

Comment:
we'd need to make this optional with a #define, and not change cache settings by default

Reply:
Fixed